tests/coprocessor/test_select.rs

```diff
@@ -266,6 +267,7 @@ fn test_select() {
 
 #[test]
 fn test_group_by() {
+    init_log();
```

Comment:
These tests will be run first. Hence all other tests' log will be printed as well.